ci/test_run_all.sh

```diff
@@ -9,6 +9,6 @@ export LC_ALL=C.UTF-8
 set -o errexit; source ./ci/test/00_setup_env.sh
 set -o errexit; source ./ci/test/03_before_install.sh
 set -o errexit; source ./ci/test/04_install.sh
-set -o errexit; source ./ci/test/05_before_script.sh
+set -o errexit; source ./ci/test/05_before_script.sh &> "/dev/null"
```

Comment:
Did you mean to leave this squelched?

Reply:
Yep, `ci/test/05_before_script.sh` output isn't that useful, so suppressed it.

Reply:
Ok.